js/gimmicks/themechooser.js

```diff
@@ -63,6 +66,15 @@
                 .attr('href', $.md.prepareLink(theme.url))
                 .appendTo('head');
         }
+        $(
```

Comment:
This `inherit`s avoids white background color in dark themes for `<pre>` and `<code>`. Text color `#DD4A68` (from prism) is a little lighter pink with better contrast in dark themes.